cmd/status/print/print.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package print
+
+// Printer defines the interface the status command expects
+// for outputting information about status of resources. Different
+// implementations allow output formats tailored to different use cases.
+type Printer interface {
+
```

Comment:
Shouldn't this have some kind of `Print` signature?

Like `Reader` has a `Read` method, etc.

Reply:
It had the `PrintUntil` function. The until part seems unnecessary, so I have renamed the function to just `Print`

Reply:
yay